CODEOWNERS

```diff
@@ -4,6 +4,22 @@
 
 *       @Fusion-Power-Plant-Framework/tsvv-14
 
+# Module responsibility
+
+*/balance_of_plant/ @CoronelBuendia @je-cook @tplobo
+*/base/ @je-cook @DanShort12 @hsaunders1904
+*/builders/ @DanShort12 @hsaunders1904 @je-cook
+*/codes/ @je-cook @ivanmaione
+*/display/ @ivanmaione @DanShort12
+*/equilibria/ @CoronelBuendia @je-cook @alexanderianblair
```

Comment:
We talked about me becoming more familiar with the equilibria module, should I be on this list?

Reply:
We did, and then @je-cook said that people were struggling with scope of concern! We can talk about this though, for sure.

Reply:
Also should probably add @ivanmaione to equilibria now with the fixed boundary solver on the way

Reply:
> We did, and then @je-cook said that people were struggling with scope of concern! We can talk about this though, for sure.

Happy to leave it for now. We can always add me in if/when I start working in that area

Reply:
Might actually be a better idea to get you on magnetostatics.. come to think of it. Spread the load a bit better